src/peergos/shared/user/SharedItemCache.java

```diff
@@ -0,0 +1,48 @@
+package peergos.shared.user;
+
+import peergos.shared.cbor.CborObject;
+import peergos.shared.cbor.Cborable;
+import peergos.shared.social.SharedItem;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class SharedItemCache implements Cborable {
+
+    private final List<SharedItem> items;
```

Comment:
We need to think about this because apps should never have access to any keys I believe, and ideally paths if possible.

Reply:
removed